apps/client/src/hoc/withTracking.js

```diff
@@ -13,7 +13,11 @@ const withTracking = (Component) => ({ ...props }) => {
       trackPageView({});
     }
   };
-  const matomoTrackEvent = ({ category, action, name }) => {
+  const matomoTrackEvent = ({ action, category, name }) => {
+    // Temporary disable Matomo trackevents
+    return;
```

Comment:
You could add a console.log and/or isProduction check here for easier testing and debugging.

Reply:
smart!